src/screens/Home/Home.js

```diff
@@ -1,19 +1,49 @@
 import React, { useState, useEffect, useCallback } from 'react';
-import { Text, View, FlatList, Image, ActivityIndicator, TouchableHighlight } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import {
+  Text,
+  View,
+  FlatList,
+  Image,
+  ActivityIndicator,
+  TouchableHighlight,
+  ImageBackground,
+} from 'react-native';
 import { useQuery } from '@tanstack/react-query';
-import { TouchableWithoutFeedback } from 'react-native-gesture-handler';
+import { ScrollView, TouchableWithoutFeedback } from 'react-native-gesture-handler';
 import { useDispatch, useSelector } from 'react-redux';
 import NetInfo from '@react-native-community/netinfo';
+import { SafeAreaView } from 'react-native-safe-area-context';
 import { styles } from '@/screens/Home/Home.styles';
 import { IMAGE_URL } from '@/controllers/routes';
 import { NAVIGATION } from '@/constants';
 import { getFavoriteMovies } from '@/selectors/MovieSelectors';
 import { networkService } from '@/networking';
 import { MovieController } from '@/controllers/MovieController';
-import { addIcon } from '@/assets';
-import { saveMovie } from '@/actions/MovieActions';
+import { addFavoriteMovieIcon } from '@/assets';
+import { saveMovie, deleteMovie } from '@/actions/MovieActions';
 import { isOneDayDiff } from '@/utils/utils';
-import { EMPTY_MOVIES, MOVIES } from '@/constants/en';
+import {
+  EMPTY_MOVIES,
+  KIDS,
+  FANTASY_MOVIE,
+  ALREADY_ADDED,
+  ACTION,
+  MOVY_ORIGINAL,
+  MY_LIST,
+  PLAY,
+  INFO,
+  ALL_MOVIES,
+} from '@/constants/en';
+
+const image_background =
+  '/Users/arielverdugo/ejerciciosReactNative/MovieAppReactNative/src/assets/img/background_home.png/';
+const icon_plus =
+  '/Users/arielverdugo/ejerciciosReactNative/MovieAppReactNative/src/assets/img/Path.png';
+const icon_play =
+  '/Users/arielverdugo/ejerciciosReactNative/MovieAppReactNative/src/assets/img/Shape.png';
+const icon_info =
+  '/Users/arielverdugo/ejerciciosReactNative/MovieAppReactNative/src/assets/img/Shape-3.png';
```

Comment:
I suggest moving these variables elsewhere (probably to a constants folder)

Reply:
`@/assets`

Reply:
Done!